### PR TITLE
SSO zstream onboarding on ibmcloud

### PR DESCRIPTION
**Pull request by smpsgill**

### CODE REVIEW

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-rhbk-interop-ocp-ci-main-rhsso-product-ocp4.15-lp-interop-rhsso-interop-ibmcloud

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-rhbk-interop-ocp-ci-main-rhsso-product-ocp4.15-lp-interop-rhsso-interop-ibmcloud

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-rhbk-interop-ocp-ci-main-rhsso-product-ocp4.15-lp-interop-rhsso-interop-ibmcloud

**Comment by Simranpal** [comment]
/pj-rehearse periodic-ci-rhbk-interop-ocp-ci-main-rhsso-product-ocp4.14-lp-interop-rhsso-interop-ibmcloud

File: ci-operator/config/rhbk/interop-ocp-ci/rhbk-interop-ocp-ci-main__rhsso-product-ocp4.14-lp-interop.yaml [hunk 1]
```diff
@@ -31,20 +31,20 @@ resources:
...
     env:
-      BASE_DOMAIN: cspilp.interop.ccitredhat.com
+      BASE_DOMAIN: zstream.interop.ccitredhat.com
       FIREWATCH_CONFIG: |
         {
           "failure_rules":
             [
               {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "group": {"name": "cluster", "priority": 1}, "jira_additional_labels": ["!default"]},
-              {"step": "gather-*", "failure_type": "pod_failure", "classification": "Other", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["!default"]},
+              {"step": "gather-*", "failure_type": "pod_failure", "classification": "Other",  ee"group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["!default"]},
```
Comment (calebevans) [inline ci-operator/config/rhbk/interop-ocp-ci/rhbk-interop-ocp-ci-main__rhsso-product-ocp4.14-lp-interop.yaml:45]:
```suggestion
              {"step": "gather-*", "failure_type": "pod_failure", "classification": "Other",  "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["!default"]},
```

File: ci-operator/jobs/rhbk/interop-ocp-ci/rhbk-interop-ocp-ci-main-periodics.yaml [hunk 2]
```diff
@@ -90,12 +90,12 @@ periodics:
...
   labels:
-    ci-operator.openshift.io/cloud: aws
-    ci-operator.openshift.io/cloud-cluster-profile: aws-cspi-qe
+    ci-operator.openshift.io/cloud: ibmcloud
+    ci-operator.openshift.io/cloud-cluster-profile: ibmcloud-cspi-qe
     ci-operator.openshift.io/variant: rhsso-product-ocp4.14-lp-interop
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-rhbk-interop-ocp-ci-main-rhsso-product-ocp4.14-lp-interop-rhsso-interop-aws
+  name: periodic-ci-rhbk-interop-ocp-ci-main-rhsso-product-ocp4.14-lp-interop-rhsso-interop-ibmcloud
```
Comment (calebevans) [inline ci-operator/jobs/rhbk/interop-ocp-ci/rhbk-interop-ocp-ci-main-periodics.yaml:98]: Do you need a slack config here?

**Review by calebevans — CHANGES_REQUESTED** [review]

File: ci-operator/jobs/rhbk/interop-ocp-ci/rhbk-interop-ocp-ci-main-periodics.yaml (hunk 2 above)
Comment (Simranpal) [reply ci-operator/jobs/rhbk/interop-ocp-ci/rhbk-interop-ocp-ci-main-periodics.yaml:98]: yeah, I will add it here. Thanks!

**Review by calebevans — APPROVED** [review]
/lgtm

**Comment by calebevans** [comment]
/pj-rehearse periodic-ci-rhbk-interop-ocp-ci-main-rhsso-product-ocp4.14-lp-interop-rhsso-interop-ibmcloud

**Comment by Simranpal** [comment]
/pj-rehearse periodic-ci-rhbk-interop-ocp-ci-main-rhsso-product-ocp4.14-lp-interop-rhsso-interop-ibmcloud

**Pull request merged**